src/index.html

```diff
@@ -4,9 +4,404 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header__logo">
+        <img src="images/logo.png" alt="Logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link"
+            data-qa="nav-hover"
+            >Laptops & Computers</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section class="catalog">
+        <div class="catalog__card" data-qa="card">
+          <img
+            class="catalog__card-img"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac"
+          >
+
+          <h2 class="catalog__card-title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="catalog__card-product-code">
+            Product code: 195434
+          </p>
+
+          <div class="stars">
+            <div class="stars__block stars__block--4">
+              <div class="stars__block-star"></div>
+              <div class="stars__block-star"></div>
+              <div class="stars__block-star"></div>
+              <div class="stars__block-star"></div>
+              <div class="stars__block-star"></div>
+            </div>
+
+            <div class="stars__reviews">
+              <p>
+                Reviews: 5
+              </p>
+            </div>
+          </div>
```

Comment:
```suggestion
          <div class="card__element-name-example">
            <div class="stars stars--4">
              <div class="stars__star"></div>
              <div class="stars__block-star"></div>
              <div class="stars__block-star"></div>
              <div class="stars__block-star"></div>
              <div class="stars__block-star"></div>
            </div>

            <div class="card__reviews">
              <p>
                Reviews: 5
              </p>
            </div>
          </div>
```
element with name stars__block looks a bit weird. Change the structure as on example (i.e. just reuse `stars` BEM block from the previous task)